the-experts-fe/src/App.js

```diff
@@ -19,9 +16,11 @@ export default function App() {
   ]);
   const [jobValue, setJobValue] = useState("");
   const [selectedContractors, setSelectedContractors] = useState([]);
-
+  
+  
   useEffect(() => {
-    fetch(`http://localhost:9000/contractor-results/${jobValue}`)
+    if(jobValue){
+      fetch(`http://localhost:9000/contractor-results/${jobValue}`)
       .then(res => res.text())
```

Comment:
Remember to use `res.json()` instead of `res.text()` here as the content-type of the response is JSON.